Discussion: NATIVE-305: Prepare for testbirds (#159)
Integreat/integreat-react-native-app: docs/test-surveys/13-survey-react-native-release.md

> ### UC1: Open the app and choose a city of your choice as a location and the language you are most comfortable with. Explore the app and familiarize yourself with it. Turn off your internet connection and explore the same city and a further on.

Reviewer: What do you mean by `a further on`?

> ### UC1: Open the app and choose a city of your choice as a location and the language you are most comfortable with. Explore the app and familiarize yourself with it. Turn off your internet connection and explore the same city and a further on.

Reviewer 2: ```suggestion
### UC1: Open the app and choose a city of your choice as a location and the language you are most comfortable with. Explore the app and familiarize yourself with it. Turn off your internet connection and explore the same city further on.
```

#### Describe the steps you took.

*Input box*

> ### UC2: Choose city "Stadt Regensburg" as your location, if you haven’t done so already. Find out when shops are usually open in Regensburg. Switch the language as soon as you found it.

Reviewer: You should probably mention the goal should be to find the information without using the search.

#### Did you find the information?

* [ ] Yes
* [ ] No

(The Answer is: "Mo-Sat: 08:00-20:00")

#### Describe the steps you took.

*Input box*

### UC3: Find the same information as in UC2 by using the search icon in the navigation bar. 

The search terms you need are "Einkaufen" (German), "Shopping" (German), "التسوّق" (Arabic/العربية)

#### Did you find the information?

* [ ] Yes
* [ ] No

> ### UC4: Choose city "Stadt Regensburg" as your location, if you haven’t done so already. Try to find a job in the "Sprungbrett into work" offer. Also try the "Lehrstellenradar".

Reviewer: Why not use a different city here (or above)?

Note: You need a stable internet connection.

> #### Did you a job offer near Regensburg?

Reviewer: ```suggestion
#### Did you find a job offer near Regensburg?
```

* [ ] Yes
* [ ] No

### UC5: Turn off your internet connection (WiFi + Data) and repeat usecases UC2-4.

> Note: If the app does show you what you require internet then it is not a bug!

Reviewer: ```suggestion
Note: If the app does show you that an internet connection is required then it is not a bug!
```
Also this is not entirely true, is it? Already opened cities/languages should work.

### UC6: Internal Linking

// TODO

### UC6: Events

// TODO

### UC7: Sharing 

// TODO

> ### UC8: Feedback 

Reviewer: Maybe a separate usecase testing the language change would make sense.

// TODO

> ### UC-FINAL: Explore the app by yourself, the more bugs you find the higher your payout will be! Check different tabs and different settings, or try some of the following actions:

Reviewer 2: What are tabs?

- Minimizing the app and using others (multitasking)

> - Change font size in the settings of your smartphone

Reviewer 2: ```suggestion
- Change font size in the system settings of your smartphone
```

- Going offline and using the app without internet connection
- Switch on Airplane Mode
- Use the app at a location with bad reception
- Change between Landscape and Portrait mode
- Change language via the globe button
- Anything else you can think of...

### Q1: How easy did you think it was to navigate yourself in the app? (1: Very easy/6: Very difficult)

* [ ] (Very Easy)
* [ ] \-
* [ ] \-
* [ ] \-
* [ ] \-
* [ ] (Very difficult)

### Q2: How easy did you think it was to find what you were looking for? (1: Very easy/6: Very difficult)

* [ ] (Very Easy)
* [ ] \-
* [ ] \-
* [ ] \-
* [ ] \-

> * [ ] (Very difficult)

Reviewer: Maybe add a question for general observations.
